templates/index.html

```diff
@@ -8,9 +8,9 @@
 
     <title>Gerrit Patch Uploader</title>
 
-    <link rel="stylesheet" href="//tools-static.wmflabs.org/static/bootstrap/3.1.1/css/bootstrap.min.css" />
+    <link rel="stylesheet" href="//tools-static.wmflabs.org/cdnjs/ajax/libs/twitter-bootstrap/3.1.1/css/bootstrap.min.css" />
     <link rel="stylesheet" href="//tools-static.wmflabs.org/static/bootstrap/tool/tools-footer.css" />
```

Comment:
I pledged to use the same versions to avoid any undesired changes in the application's behavior, but they can be easily upgraded if @valhallasw wishes.